index.html

```diff
@@ -1,114 +1,214 @@
+        máshol keresi az utópiáját. Van, aki az italboltban vagy a bankban,
+        ahogy azt a Utopia-óriásplakátok ironikusan hirdették, van, aki pedig
+        Drake hotelszobájában.
+      </p>
+      <p>
+        Ahogy az az After adásában is felmerül, a közvélemény a Utopia
+        megjelenése óta arról beszél, hogy ez a lemez lényegében a Yeezus
+        második kiadása, Kanye West hatása pedig olyannyira elvitathatatlan,
+        hogy az új lemez már-már a 2013-as album másolata. A valós kép ennél
+        árnyaltabb:
+        <span class="highlight"
+          >West producerkedett a Thank God című dalon, a Telekinesist és God’s
+          Countryt pedig eredetileg a 2021-es Dondára szánták</span
+        >, a számokat viszont Travis Scott kapta kézhez befejezésre, amolyan
+        kivételezett örökösként. A legérdekesebb pedig a kulcsdalnak számító
+        Modern Jam sztorija. A számhoz a Yeezusről származó I Am a God egyik
+        eredeti, tíz évvel ezelőtti verzióját használták újra, amit még 2013-ban
+        együtt rakott össze Travis Scott, Kanye West, Mike Dean és a Daft Punk.
+      </p>
+      <p>
+        Nem mindegy hát, hogy melyik fiókba rakjuk a rapper-producert. Kanye
+        Westnek ugyanis vannak protezsáltjai, mint az idei Szigeten is fellépő
+        070 Shake, és vele közel egyenértékű kollégái és partnerei is. Travis
+        sokkal inkább az utóbbi, aki így nem másol, hanem átvesz és végre
+        megfőzi azt, aminek a receptjét anno ő is papírra vetette.
+      </p>
+      <p>
+        Különleges státuszát kihasználva minderre csak ráerősített a Utopia
+        lemez első élő előadása alatt augusztus 7-én a római Circus Maximus
+        romjai közt, amikor meglepetésvendégként <span class="highlight">színpadra hívta</span> az
+        antiszemita botrányát is túlélő (?) <span class="highlight">Westet</span>, majd így szólt:
+        „Kanye West nélkül nem lenne Utopia. Kanye West nélkül Travis Scott
+        sincs.”
+      </p>
+      <p>Az adásban szóba kerül még:</p>
+      <ul>
+        <li>Meg tudott-e újulni Travis Scott az évek során?</li>
+        <li>
+          Rendben van-e a Kanye Westtel való haverkodás az előadó náci
+          megnyilvánulásai után?
+        </li>
+        <li>Mi a Travis Scott-recept sikere?</li>
+        <li>Producerként vagy rapperként erősebb Travis Scott?</li>
+        <li>Sikerült-e kimozdítania közreműködőit a komfortzónájukból?</li>
+        <li>Ki a legmeglepőbb közreműködő a lemezen?</li>
+        <li>Önreflexív album tud-e lenni a Utopia?</li>
+        <li>
+          Ideális, tökéletes jövőképet fest a rapper, vagy mindez csak egy
+          trükk?
+        </li>
+      </ul>
+    </article>
+    <div class="form">
+      <form name="Cikk küldése ismerősnek" action="" autocomplete="on">
+        <fieldset>
+          <legend>Cikk küldése ismerősnek</legend>
+          <label for="name">*Név:</label
+          ><input type="text" id="name" name="name" required />
```

Comment:
Néha az autoformázó nem tudja eldönteni, hogy mikor kell új sorba tennie bizonyos elemeket. Most pl. ez a záró relációsjel az első karakter ebben a sorban. Tipp: ha visszateszed az előző sorba és az input-ot te magad teszed új sorba, az autoformázó nem fogja újra-elrontani. Én is küzdök vele minden nap :)